### PR TITLE
Multifractal2

### PR DESCRIPTION
**Pull request by DimpyRed**

### CODE REVIEW

File: client/src/worldgen.rs [hunk 1]
```diff
@@ -124,15 +125,40 @@ pub fn voxels(graph: &DualGraph, node: NodeId, chunk: Vertex) -> VoxelData {
...
-                    r if r < 1.0 => (Material::Stone, -1.0),
-                    _ => (Material::Sand, -0.5),
-                };
+                let elev = trilerp(&enviros.max_elevations, cube_coords);
+                let rain = trilerp(&enviros.rainfalls, cube_coords);
+                let temp = trilerp(&enviros.temperatures, cube_coords);
+                let slope = trilerp(&enviros.slopeiness, cube_coords);
+
+                let mut voxel_mat = Material::Dirt;
+                let max_e = elev;
```
Comment (Ralith) [inline client/src/worldgen.rs:134]: I don't think we need two different variables for this. Just replace use of max_e with elev?

File: common/src/world.rs [hunk 2]
```diff
@@ -9,10 +9,17 @@ pub enum Material {
     Sand = 3,
     Wood = 4,
     Leaves = 5,
+    Water = 6,
+    Snow = 7,
+    Grass_top = 8,
```
Comment (Ralith) [inline common/src/world.rs:14]:
```suggestion
    Grass = 8,
```

File: client/src/worldgen.rs (hunk 1 above)
Comment (DimpyRed) [reply client/src/worldgen.rs:134]: I disagree. Max_e will eventually be modified by other factors.

Comment (DimpyRed) [reply client/src/worldgen.rs:134]: Max_e is an ouput, elev is an input.

**Pull request merged**